mllib/src/test/scala/org/apache/spark/ml/fpm/FPGrowthSuite.scala

```diff
@@ -103,6 +103,22 @@ class FPGrowthSuite extends SparkFunSuite with MLlibTestSparkContext with Defaul
       FPGrowthSuite.allParamSettings, checkModelData)
   }
 
+  test("SPARK-19940 - FPGrowth prediction should not contain duplicates") {
```

Comment:
This may be a violation of style. Not sure if we need the jira id here as this is self-explanatory.